qfieldsync/ui/package_dialog.ui

```diff
@@ -111,11 +81,33 @@
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
-          <string>Export Directory</string>
+          <string>Packaged Project Title</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
-           <widget class="QLineEdit" name="manualDir"/>
+           <widget class="QLineEdit" name="packaged_project_title"/>
```

Comment:
```suggestion
           <widget class="QLineEdit" name="packagedProjectTitleLineEdit"/>
```
Note that the names here are camelCase for historical reasons. Don't mix multiple naming conventions.